### ISSUE TITLE
[Feature] Aqara Smart plug (EU) - resolve state after power loss

### ISSUE THREAD

**Issue by jaccostraaijer** [TedTolboom/com.xiaomi-mi-zigbee#230]
Is it  possible that after a power loss the Aqara Smart plug (EU) initiates the same state (ON or OFF) as before power was lost?
(I have seen this feature with the Fibaro smart plugs on Homey)

**Comment by epjjansen** [comment]
Is this not being accomplished by the "Always On" setting in the advanced settings?

**Comment by jaccostraaijer** [comment]
No, always on means you cannot turn the plug in off state. What I mean is that after a power cut the last state is being recovered.

**Comment by TedTolboom** [comment]
@jaccostraaijer I can't find an option (attribute) to enable this from device side; also checked it at other opensource platforms, but this feature is not present in the devices firmware.

I'll close this issue; if you happen to find additional information how to implement this, please let me know

**Comment by TedTolboom** [comment]
And re-opened

**Comment by TedTolboom** [comment]
There might be an option

**Comment by epjjansen** [comment]
I have a kind of work around for this
A flow run every <period> then turn powerplugs on. In a case when power was lost and homey runs again plugs are turned on again. Most kitchen appliances like refrigerators will keep their contents chilled/frozen for 24 hrs in the event of a power outage
